src/filing/App.jsx

```diff
@@ -99,6 +99,10 @@ export class AppContainer extends Component {
   }
 
   isValidPeriod(period) {
+    // TODO: Account for past periods/use derived period options here
```

Comment:
Based on feedback during standup, all available filing periods will remain the config, so this change won't be required.